pkg/constants/constants.go

```diff
@@ -90,3 +103,17 @@ const (
 	KlusterletSuffix     = "klusterlet"
 	KlusterletCRDsSuffix = "klusterlet-crds"
 )
+
+const (
+	// ConditionManagedClusterImportSucceeded is the condition type of managed cluster to indicate whether the managed
+	// cluster is imported successfully
```

Comment:
should we consider to move the condition definitions  to the [clusterlifecycle-api](https://github.com/stolostron/cluster-lifecycle-api) ? if there are other components depend on these conditions?

Reply:
ok, will submit a follow up PR to do this.